controllers/snapshot/snapshot_adapter.go

```diff
@@ -285,6 +286,24 @@ func (a *Adapter) createMissingReleasesForReleasePlans(releasePlans *[]releasev1
 				"Application.Name", a.application.Name,
 				"ReleasePlan.Name", releasePlan.Name,
 				"Release.Name", newRelease.Name)
+			//Copy PipelineRun PAC annotations/labels from applicaitonSnapshot to Release
```

Comment:
The copying of labels and annotations should happen before the `a.Client.Create(a.context, newRelease)` call on line 281 in order for them to be present on the created object.

Reply:
Updated.